application_form/api/views.py

```diff
@@ -40,3 +41,25 @@ def get(self, request, project_uuid):
         )
         serializer = self.get_serializer(reservations, many=True)
         return Response(serializer.data)
+
+
+class LatestApplicantInfo(GenericAPIView):
+    """
+    Public: Returns the primary applicant from the latest application.
+    """
+
+    serializer_class = ApplicantSerializer
```

Comment:
This will include SSN (and birth date) into the response, but it is not used in any way in the front end AFAIK. It would be best to not expose personal information that is not needed, so a separate serializer for this view  would be nice.